.trybe/requirements.json

```
@@ -1,51 +1,51 @@
 {
   "requirements": [
     {
-      "description": "A página deve possuir o título \"Paleta de Cores\".",
+      "description": "1 - Adicione à página o título \"Paleta de Cores\".",
       "bonus": false
     },
     {
-      "description": "A página deve possuir uma paleta de quatro cores distintas.",
+      "description": "2 - Adicione à página uma paleta de quatro cores distintas.",
       "bonus": false
     },
     {
-      "description": "A cor preta deve ser a primeira na paleta de cores.",
+      "description": "3 - Adicione na página a cor **preta** ela deve ser a primeira na paleta de cores.",
```

Achei que a frase estava um pouco confusa. Faz sentido?
```suggestion
      "description": "3 - Adicione a cor **preta** como a primeira cor da paleta de cores.",
```